pkg/genevalogging/genevalogging.go

```diff
@@ -322,12 +240,7 @@ func (g *genevaLogging) CreateOrUpdate(ctx context.Context) error {
 
 	g.log.Print("waiting for privileged security context constraint")
 	timeoutCtx, cancel := context.WithTimeout(ctx, 30*time.Minute)
-	defer cancel()
-	var scc *securityv1.SecurityContextConstraints
-	err = wait.PollImmediateUntil(10*time.Second, func() (bool, error) {
-		scc, err = g.seccli.SecurityV1().SecurityContextConstraints().Get("privileged", metav1.GetOptions{})
-		return err == nil, nil
-	}, timeoutCtx.Done())
```

Comment:
also this could be pulled out to an install condition at some point

Reply:
With @hawkowl's work moving `pkg/install` -> `pkg/cluster` does it makes sense to wait to do this? @jim-minter @asalkeld @mjudeikis ? 

Fixing the `pkg/util/ensure` will alleviate a lot of the duplicated logic with the daemonsets at least, but I don't want to complicate the installer.

Reply:
Also, I need the privileged SCC returned to create a new one. I could create a function that creates the resulting SCC object for all the functions (just passing in the params of what's overridden in this CreateOrUpdate to leave the Interface to only handle "Ensuring" the k8s objects are created or updated.

Reply:
@troy0820 the aro-operator code will also rework this.. why not hold off for a bit?